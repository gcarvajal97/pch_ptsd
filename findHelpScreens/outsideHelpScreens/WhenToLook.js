import React, { Component } from 'react';
import { View, StyleSheet, Text} from 'react-native';
import { translate } from '../../components/translateService';
import colors from '../../colors';


class WhenToLook extends Component {

  render() {
    return (
      <View>
        <Text style={styles.paragraphTitle}>{translate('whenToLook.paragraphTitle1')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph1')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph2')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph3')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph4')}</Text>
        <Text style={styles.paragraphTitle}>{translate('whenToLook.paragraphTitle2')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph5')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph6')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph7')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph8')}</Text>
        <Text style={styles.paragraph}>{translate('whenToLook.paragraph9')}</Text>
    </View>
    );
  }
}

export default WhenToLook;

const styles = StyleSheet.create({
  paragraphTitle: {
    fontFamily: "avenir-heavy",
    fontSize: 20,
    alignSelf: 'stretch',
    textAlign: 'left',
    marginTop: 6,
    marginBottom: 3,
    color: colors.accent
  },
  paragraph: {
    textAlign: 'left',
    fontSize: 14,
    marginVertical: 2
  },
  paragraphBold: {
    textAlign: 'left',
    fontSize: 14,
    fontFamily: "avenir-heavy",
    marginVertical: 2,
    marginLeft: 10
  },
  link: {
    fontSize: 14,
    color: colors.accent,
    textDecorationLine: 'underline'
  }
});