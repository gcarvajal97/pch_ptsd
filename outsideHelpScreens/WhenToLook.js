import React, { Component } from 'react';
import { View, StyleSheet, Text, ScrollView, Dimensions, AccessibilityInfo, Image} from 'react-native';

class WhenToLook extends Component {

  render() {
    return (
      <View>
        <Text style={styles.title}>
        DOES MY CHILD NEED COUNSELING OR ADDITIONAL HELP? WHEN TO LOOK FOR MORE HELP
        </Text>
        <Image 
          style={styles.image}
          source={require('../assets/WhenToLookImage.jpg')}
        />
        <Text style={styles.paragraphTitle}>
        Does your child have traumatic stress symptoms that:
        </Text>
        <Text style={styles.bullet}>
        {String.fromCharCode(0x2022)} Get in the way of getting back to usual activities?
        </Text>
        <Text style={styles.bullet}>
        {String.fromCharCode(0x2022)} Stop your child from enjoying things he used to enjoy?
        </Text>
        <Text style={styles.bullet}>
        {String.fromCharCode(0x2022)} Seem to be getting worse rather than better?
        </Text>
        <Text style={styles.bullet}>
        {String.fromCharCode(0x2022)} Have lasted more than a month?
        </Text>
        <Text style={styles.paragraphTitle}>
        Does your child:
        </Text>
        <Text style={styles.bullet}>
        {String.fromCharCode(0x2022)} Get physical symptoms if reminded of the injury or what happened?
        </Text>
        <Text style={styles.bullet}>
        {String.fromCharCode(0x2022)} Try not to talk about it?
        </Text>
        <Text style={styles.bullet}>
        {String.fromCharCode(0x2022)} Startle easily (for example, jumping at sudden noises)?
        </Text>
        <Text style={styles.bullet}>
        {String.fromCharCode(0x2022)} Get very distressed if reminded of the injury?
        </Text>
        <Text style={styles.paragraph}>
        If any of the above are true, it's a good idea to talk 
        with your doctor or another professional about 
        how your child is doing.
        </Text>
    </View>
    );
  }
}

export default WhenToLook;

const styles = StyleSheet.create({
  container: {
      flex: 1,
      alignItems: 'center',
      margin: 10,
      backgroundColor: 'white',
      shadowColor: "#000",
      shadowOffset: {
          width: 0,
          height: 2,
      },
      shadowOpacity: 0.23,
      shadowRadius: 2.62,

      elevation: 4,
  },
  title: {
      backgroundColor: '#2089DC', 
      color: 'white', 
      fontWeight: 'bold', 
      fontSize: 22, 
      alignSelf: 'stretch'
  },
  subTitle: {
    backgroundColor: '#2089DC', 
    color: 'white', 
    alignSelf: 'stretch', 
    paddingLeft: 15, 
    paddingRight:15, 
    fontSize: 15, 
    borderBottomWidth:2, 
    borderBottomColor:'#2089DC',
  },
  paragraphTitle: {
      fontWeight: 'bold',
      fontSize:20,
      alignSelf:'stretch',
      textAlign:'left',
      marginLeft:5,
      marginTop:4,
      marginBottom:3,
      color:'#2089DC'
  },
  bullet: {
      marginVertical:1, 
      marginHorizontal:18, 
      textAlign:'left', 
      alignSelf:'stretch', 
      fontWeight:'bold',
      fontSize:14
  },
  paragraph: {
      alignSelf:"center", 
      paddingVertical:1, 
      paddingHorizontal:5,
      fontSize: 14,
      marginTop: 5,
  },
  link: {
      alignSelf:"center", 
      paddingVertical:1, 
      paddingHorizontal:5,
      fontSize: 16,
      color:'#2089DC',
      textDecorationLine: 'underline',
      marginTop: 4,
      marginTop: 3,
  },
  image: {
    flex: 1,
    aspectRatio: 1.525,
    resizeMode: 'contain'
  }
});