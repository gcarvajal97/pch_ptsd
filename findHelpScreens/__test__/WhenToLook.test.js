import React from 'react';
import WhenToLook from '../outsideHelpScreens/WhenToLook';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<WhenToLook/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 