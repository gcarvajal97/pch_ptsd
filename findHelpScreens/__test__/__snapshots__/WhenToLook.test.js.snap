// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Does your child have traumatic stress symptoms that:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Get in the way of getting back to usual activities?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Stop your child from enjoying things they used to enjoy?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Seem to be getting worse rather than better?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Have lasted more than a month?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Does your child:
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Get physical symptoms if reminded of the injury or what happened?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Try not to talk about it?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Startle easily (for example, jumping at sudden noises)?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    -Get very distressed if reminded of the injury?
  </Text>
  <Text
    style={
      Object {
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    If any of the above are true, it's a good idea to talk with your doctor or another professional about how your child is doing.
  </Text>
</View>
`;
